src/backends/automate.js

```diff
@@ -155,8 +153,6 @@ export default class AutomateBackend extends BaseBackend {
 
         var sessionId = this.sessions[id].sessionId;
 
-        this.sessions[id].interval = setInterval(() => requestApi(BROWSERSTACK_API_PATHS.getUrl(sessionId), { executeImmediately: true }), API_POLLING_INTERVAL);
```

Comment:
We can't remove it, since TestCafe doesn't use Selenium commands for test actions, and BrowserStack can terminate TestCafe's session as an idle session. 

https://www.browserstack.com/automate/timeouts

Reply:
It will terminate only if there is no command being sent to browserstack, but as commands are immediately sent tests are working fine on browserstack.

Reply:
Also adding an extra URL pooling with setTimeout puts the method in the event queue and decreases the performance of the binary.

Reply:
Please run a big TestCafe suite and check Text Logs in the BrowserStack dashboard. It contains only one command that we execute with this `setInterval`.

![image](https://user-images.githubusercontent.com/4479386/67082046-035d8500-f1a1-11e9-898c-f87b1339d6a6.png)
![image](https://user-images.githubusercontent.com/4479386/67082114-24be7100-f1a1-11e9-86c9-8638aadba209.png)

Without it, there will be no Selenium commands sent to the BrowserStack server, and the session will be terminated.

Reply:
Nope, it is working fine and it is only sending unnecessary fetch url actions in event queue.

Reply:
It doesn't. Make sure that the Automation backend is enabled by checking the `BROWSERSTACK_USE_AUTOMATE` environment variable and setting it to `1`.

![image](https://user-images.githubusercontent.com/4479386/67194321-8d0a8e00-f3ff-11e9-9606-77948b3bb978.png)

Reply:
Not happening on my end, ran multiple sessions and also the example given in testcafe repo. Working fine though.

Reply:
![image](https://user-images.githubusercontent.com/18033231/67261202-48ddc300-f4bd-11e9-831a-588a997ebefd.png)

Also, I have some concerns around explicit waits and not using philosophy executeImmediately which you have used.

Reply:
1m 12s is 72 seconds. The idle timeout is 90 seconds.

Reply:
It's the same example that you are running for demonstrating error. And BTW my concerns are on the explicit delay that you have provided and `execute immediatedly` stuff